integrationTests/vm/esdt/esdtProcess_test.go

```diff
@@ -879,14 +879,14 @@ func TestScCallsScWithEsdtCrossShard(t *testing.T) {
 
 	nonce, round = integrationTests.WaitOperationToBeDone(t, nodes, 4, nonce, round, idxProposers)
 	_, err = nodes[2].AccntState.GetExistingAccount(firstScAddress)
-	assert.Nil(t, err)
+	require.Nil(t, err)
 
 	//// call first sc with esdt, and first sc automatically calls second sc
 	valueToSendToSc := int64(1000)
 	txData := core.BuiltInFunctionESDTTransfer + "@" +
```

Comment:
why not use the new fancy txData.New()  ? :D

Reply:
Added.